tests/MockDataTest/scripts/run_test.sh

```
@@ -43,6 +43,10 @@ if [[ -z "$EMOTIBIT_CSV" ]]; then
     exit 1
 fi
 
+cd "$PROJECT_ROOT/build/Release"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)